app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -80,11 +80,20 @@ def facilities
     private
 
     def active_facilities(lighthouse_facilities)
```

Comment:
I know this _should_ only happen in an environment where the import job hasn't been run yet, but should we put this behind a toggle just to make sure we can turn it off in prod for whatever reason? Or does that not play nice with the RI's? I'm not sure what those  ones get by default. So it may not be a big deal.

Reply:
You mean a temporary toggle to make sure the release goes smoothly, or a longer-term one?
I think a release toggle would be fine - always nice to decouple deploy from activation. But a permanent one would work at cross-purposes here - the hope is to make this not require manual effort for a new environment, and a permanent toggle would mean you have know to enable it in the RI every time, right? If you're doing that, you're not saving as much over knowing you have to SSH in and run the job manually, IMHO - the % difference between 0 and 1 steps feels larger than the difference between 1 and 5 steps?

Reply:
Yeah, just a temporary toggle for the release process. I agree on not using a long term toggle 👍

Reply:
And that's only if the toggle works nice with the RIs. If they don't then not using one is fine by me.

Reply:
yep, they work there, the only issue is you have to manually turn them on.